code/game/machinery/vending.dm

```diff
@@ -1561,14 +1566,17 @@
 					/obj/item/part/armor = 700,
 					/obj/item/part/gun = 700,
 					/obj/item/gun/projectile/automatic/ak47/fs = 3200,
-					/obj/item/gun/energy/retro = 2000,
-					/obj/item/gun/projectile/mk58  = 1500,
-					/obj/item/gun/projectile/mk58/wood = 1500,
+					/obj/item/gun/energy/retro = 1200,
+					/obj/item/gun/projectile/shotgun/doublebarrel = 1400,
+					/obj/item/gun/projectile/mk58  = 900,
+					/obj/item/gun/projectile/mk58/wood = 900,
+					/obj/item/gun/projectile/colt = 1200,
 					/obj/item/gun/projectile/mandella = 1800,
 					/obj/item/gun/projectile/revolver/deckard = 3600,
 					/obj/item/gun/projectile/automatic/z8 = 3500,
-					/obj/item/gun/projectile/automatic/molly = 2000,
+					/obj/item/gun/projectile/automatic/molly = 1700,
 					/obj/item/gun/projectile/shotgun/pump/gladstone = 2200,
+					/obj/item/gun/projectile/shotgun/pump/regulator = 1900,
```

Comment:
The regulator is the strongest shotgun in the game, shouldn't be this cheap.

Reply:
> The regulator is the strongest shotgun in the game, shouldn't be this cheap.

Exactly

Reply:
Should it be more than gladstone?